server/src/aiController.js

```diff
@@ -0,0 +1,54 @@
+const OpenAI = require('openai');
+
+const ai = async (request, response) => {
+  const { theme, quantity, gender } = request.body;
+  const openai = new OpenAI({
+    organization: 'org-DhkAl70aE8X3otKSx9c7HKU7',
+    apiKey: `${process.env.OPENAI_KEY}`,
+    dangerouslyAllowBrowser: true,
+  });
+
+  try {
+    const result = await openai.chat.completions.create({
+      messages: [
+        {
+          role: 'system',
+          content: `You are a creative assistant who generates creative and unique pet names.
+            Return the response as a JSON object with a key of names and the values in an array object.`,
+        },
+        {
+          role: 'user',
+          content: `create ${quantity} random ${gender} pet names with a ${theme} theme.  
+          Return ${quantity} pet names.
+          Return only the names.  
+          Never return a same name from the previous response.
+          Never return more than five names.
+          If the theme is not words return a value of null`,
```

Comment:
I like the specificity of the prompts. I haven't done any prompt engineering yet, but what have you tried feeding it nonsense themes? Is it smart enough to return null or can you filter those out?

Reply:
most of the time it still returns silly names.  if you enter random characters it will return similar random characters but in a different order.  I will need to handle that scenario (if able).